_std/macros/voice.dm

```diff
@@ -0,0 +1,2 @@
+//human voice stuff
+#define RANDOM_HUMAN_VOICE pick(1,2,3)
```

Comment:
I'd rather see this in a file named `preferences.dm`  or something like that since it's a bit less specific and that's where this is used.

Reply:
good call, moved it to already existing defines/preferences.dm